src/views/GameView.vue

```diff
@@ -46,8 +43,42 @@ let teamAlpha = ref<Array<string>>([]);
 let teamBeta = ref<Array<string>>([]);
 const playerName = ref("");
 
+function connectToLobby() {
+  const socket = new SockJS("/minigames/towercrush/api/v1/connect");
+  stompClientGame.value = Stomp.over(socket);
+  stompClientGame.value.connect({}, () => {
+    handleMessageReceipt("Connected");
+    stompClientGame.value.subscribe(
+      "/topic/lobbies/" + lobbyName.value,
+      function (messageOutput: any) {
+        handleMessageReceipt(messageOutput.body);
+      }
+    );
+  });
+}
+
+function joinTeam(teamName: string) {
+  stompClientGame.value.send(
+    "/ws/lobby/${lobbyName.value}/join/team/${teamName}/player/${playerName.value}"
```

Comment:
Wait, I made a mistake:
```suggestion
    `/ws/lobby/${lobbyName.value}/join/team/${teamName}/player/${playerName.value}`
```
Otherwise no string interpolation will be performed.

Reply:
ok